css/index.css

```diff
@@ -1 +1,435 @@
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 8.5% 0 3.9% 0;
+   padding: 3.9% 0;
+   line-height: 1.8rem;
+}
+
+.main-content .top-content {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 3.4% auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-content .bottom-content p {
+    padding-right: 2.2%;
+}
+
+/* Contact Styles */
+
+.contact p {
+	margin: 1.7% 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 4.5% 0 2.2%;
+}
+@media screen and (max-width: 1100){
+
+	.container {
+    	width: 100%;
+    	margin: 0 auto;
+	}
+
+	header{
+		display:flex;
+		text-align:center;
+        width: 100%;
+        
+	}
+
+	header .logo {
+    	margin: 28px 0 0;
+	}
+
+
+
+	.cta {
+    	display: flex;
+    	margin-top: 5%;
+    	justify-content: center;
+        width: 100%;
+	}
+
+	.cta .cta-text {
+    	font-size: 4rem;
+    	text-align: center;
+    	display: flex;
+    	justify-content: center;
+    	align-items: center;
+    	margin: 0px 0 0;
+	}
+
+	.cta img{
+		width: 41.8%;        
+        height: 100%;
+	}
+
+	.main-content {
+    	font-size: 1.5rem;
+        line-height: 1.7rem;
+    	padding:20px 10px 20px 10px;
+    	width:100%;
+
+	}
+	.main-content .top-content {
+		flex-direction: column;
+		align-content:center;
+		height:30%;
+	}
+
+	.main-content .top-content .text-content{ 
+		height: 100%;
+		text-align: center;
+		margin-top: 3%; 
+	}
+
+	.main-content .bottom-content{
+		flex:display;
+		flex-direction: column;
+		align-content:center;
+		height:30%;
+	}
+	.main-content .bottom-content .text-content{ 
+		text-align: center;
+		margin-top: 3%; 
+
+	}
+
+	.main-content .middle-img {
+    	display: block;
+    	margin: 4% auto;
+    	width: 100%;
+	}
+
+	.contact{
+		width:100%;
+		text-align: center; 
+	}
+
+	footer{
+		width:100%;
+		margin-top: 4%;
+		border-top: 0.1px solid black;
+		padding-top: 1.2%;
+	}
+}
+
+
+
+
+
+
+/* 800px screen size ........*/
+@media screen and (max-width: 800px){
+
+	.container {
+    	width: 100%;
+    	margin: auto;
+	}
+
+	header{
+		display:flex;
+		flex-direction:column-reverse;
+		align-items:center;
+		text-align:center;
+        width: 100%;
+	}
+
+	header .logo {
+		display:flex;
+		align-self:center;
+    	margin: 28px 0 0;
+	}
+
+    .cta {
+        justify-content: center;
+    }
+
+    .cta .cta-text button {
+        font-size: 1.1rem;
+    }
+
+    .cta img {
+		display: none;
+	}
+
+	.main-content {
+    	font-size: 1.5rem;
+    	line-height: 20px;
+    	padding: 2.2%;
+    	width:100%;
+
+	}
+	.main-content .middle-img {
+		display: block;
+		margin: 3.4% auto;
+	}
+}
+
+
+/* 400px screen size ........*/
+@media screen and (max-width: 400px){
```

Comment:
Great job adding your media queries! I see you chose to use 400px here. Why 400px instead of 500px?